frontend/src/scenes/insights/aggregationAxisFormat.ts

```diff
@@ -53,6 +53,7 @@ export const canFormatAxis = (chartDisplayType: ChartDisplayType | undefined): b
             ChartDisplayType.ActionsBarValue,
             ChartDisplayType.ActionsTable,
             ChartDisplayType.WorldMap,
+            ChartDisplayType.ActionsPie,
```

Comment:
BTW looking at the UI now, I think that the axis _can_ be formatted everywhere where the "Unit" selector is shown – so do we still need `canFormatAxis()`?

Reply:
Ah ha! Yes! This is the final (brace yourself) piece of the pie. Good spot 🥇

Reply:
🤔 Does it make sense on stickiness ?

Reply:
No... I'll follow-up so it's only show if type is trends

Reply:
https://github.com/PostHog/posthog/pull/11143